lib/linter-rust.coffee

```diff
@@ -36,8 +36,8 @@ class LinterRust
             text: match.message
             filePath: if path.isAbsolute match.file then match.file else path.join curDir, match.file
             range: [
-              [match.line - 1, 0],
-              [match.line - 1, 0]
+              [match.from_line - 1, match.from_col - 1],
+              [match.to_line - 1, match.to_col - 1]
             ]
```

Comment:
you don't need a `-1` on the end col. cause in case the start and end col is the same, it won't highlight anything.